README.md

```diff
@@ -1,34 +1,34 @@
+Huron is intended as a one-stop shop for generating both an in-browser styleguide and a series of in-browser prototypes. Huron utilizes a custom CLI to process Knyle Style Sheets [(KSS)](http://warpspire.com/kss/) documentation and webpack to build the styleguides and prototypes. In addition, Huron makes heavy use of webpack's Hot Module Reloading [HMR](https://webpack.github.io/docs/hot-module-replacement.html) system to provide a quick and seamless development experience.
 
 ## Table of contents
 This file contains basic information on Huron installation and writing prototypes. However, there are several other subsections of this documentation:
 
- * [Configuration](config/README.md) - Configuration for huron and for your local webpack setup
- * [Templates](templates/README.md) - Peripheral template documentation
- * [CLI](src/cli/README.md) - General information on the Huron CLI. Source code is further documented via jsdoc
- * [Web](src/web/README.md) - General infomration on browser script for inserting/replacing markup. Source code is further documented via jsdoc
- * [Dist](dist/README.md) - General information on the distribution (build) directory.
+* [Configuration](config/README.md) - Configuration for Huron and for your local webpack setup
+* [Templates](templates/README.md) - Peripheral template documentation
+* [CLI](src/cli/README.md) - General information on the Huron CLI. Source code is further documented via jsdoc
+* [Web](src/web/README.md) - General information on browser script for inserting/replacing markup. Source code is further documented via jsdoc
+* [Dist](dist/README.md) - General information on the distribution (build) directory.
 
 ## Installation
- * Huron is not yet registered in NPM, so for now you'll have to check it out add it to your `package.json` via `file:./huron` or use a github link.
- * Don't forget to run `npm install` once you're done!
- * Once `npm install` finishes, you'll need to write a configuration or use the default. The default settings and how to modify them is documented in the configuration section of this readme.
- * Start the CLI using the `dist/cli/huron-cli.js` file.
- * Once the CLI has started, if you didn't run the CLI with the `--production` flag, you can access your prototype(s) at `localhost:[huron.port]/[huron.root]/[prototype-name].html`. You can find the defaults for the `port` and `root` options and how to change them in the config documentation, but using the defaults this URL would resolve to `localhost:8080/dist/index.html`.
+* Huron can be installed via [NPM](https://www.npmjs.com/package/huron). Add Huron to your `package.json` file.
+* Don't forget to run `npm install` once you're done!
```

Comment:
@pattiereaves Yeah, I did. Although, it's possible that I'm missing something. I'm still working on getting it up and running.